frontend/src/lib/api/whoami.ts

```diff
@@ -0,0 +1,13 @@
+import { get } from "../request";
+
+interface WhoamiResponse {
```

Comment:
Make sure to remove these old files :)

Reply:
forgot to add them to git -> will do now